repos/system_upgrade/common/actors/addupgradebootentry/libraries/addupgradebootentry.py

```diff
@@ -9,7 +9,8 @@
 
 def add_boot_entry(configs=None):
     debug = 'debug' if os.getenv('LEAPP_DEBUG', '0') == '1' else ''
-
+    enable_network = os.getenv('LEAPP_INITRAM_NETWORK') in ('network-manager', 'scripts')
+    ip_arg = ' ip=on rd.neednet=1' if enable_network else ''
```

Comment:
I prefer / have seen more often: `ip=dhcp` (also is more explicit on what is supposed to happen = will not change behaviour).

Reply:
Thanks, I will change `ip=on` to `ip=dhcp`. Thanks!